### PR TITLE
docs: fix code block formatting for `usePreviewMode`

### PR DESCRIPTION
**Pull request by florian-lefebvre**
### 🔗 Linked issue

N/A

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Currently on https://nuxt.com/docs/api/composables/use-preview-mode the code snippet is broken because badly formatted. This PR fixes this minor issue

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have added tests (if possible).
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by logotip4ik** [comment]
@florian-lefebvre  can you also update `::alert{icon=👉}` to `::note` on line 44 :pray: ?

**Comment by florian-lefebvre** [comment]
Done!

File: docs/3.api/2.composables/use-preview-mode.md [hunk 1]
```diff
@@ -40,7 +41,7 @@ const { enabled, state } = usePreviewMode({
 })
 ```
 
-::alert{icon=👉}
+::note{icon=👉}
```
Comment (DamianGlowala) [inline docs/3.api/2.composables/use-preview-mode.md:44]:
```suggestion
::note
```

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
Thank you! ❤️

**Pull request merged**